### PR TITLE
Vehicle Elevator

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Makes the elevator start raised. Fewer meme suicides.

# Explain why it's good for the game
Suicide bad.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: Vehicle Elevator now starts raised.
/:cl:

### CODE REVIEW

**Review by harryob — APPROVED** [review]

**Review by mullenpaul — APPROVED** [review]
I would prefer if the Almayer stationary dock and the admin level stationary docks are both subclasses. It would require a map change and isn't strictly necessary but it would be a cleaner solution.

**Review by mullenpaul — APPROVED** [review]
Code wise, perfect. This is the standard that I believe docking ports should be done.

File: code/modules/shuttle/vehicle_elevator.dm [hunk 1]
```diff
@@ -43,14 +43,18 @@
...
-	id = "almayer vehicle"
+	name = "Root Vehicle Elevator Dock"
+	id = "root vehicle"
 	width = 5
 	height = 5
 	dwidth = 2
 	dheight = 2
 
+/obj/docking_port/stationary/vehicle_elevator/almayer
+	name = "Alamyer Vehicle Elevator Dock"
```
Comment (harryob) [inline code/modules/shuttle/vehicle_elevator.dm:54]: alamyer

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/vehicle_elevator.dm (hunk 1 above)
Comment (realforest2001) [inline code/modules/shuttle/vehicle_elevator.dm:54]:
```suggestion
	name = "Almayer Vehicle Elevator Dock"
```

**Pull request merged**